wikimedia/wikipedia-ios
# Update displaying logic for the explore feed notifications card

**Phabricator:**  https://phabricator.wikimedia.org/T288693

### Notes
* Business logic for displaying the new Notifications Center feed card

### Test Steps
1. Log into your account (updating from a version pre-notification center)
2. You should see the old notifications feed card - the new card UI is on a separate PR
